devices/Button/ButtonBase.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
```

Comment:
Missing headers. Please check on the other devices and add cs it in all files.

Reply:
Thank you, should be resolved now